### ISSUE TITLE
App not passing props to child

### ISSUE THREAD

**Issue by caynan** [stylesuxx/generator-react-webpack-redux#4]
Inside a clean generated project, if I use `console.dir(this.props)` inside the `Main.js` file, in the console I can only see an empty Object, and given the way Main is called inside `App.js` I assume it should at least has a prop named 'actions'.

I'm sorry if this is a noob question, but I simply can't figure out what is happening.

**Comment by stylesuxx** [comment]
That is ritght, the props object should have and actions prop. Just scaffolded a clean project and checked that this is true => 
![image](https://cloud.githubusercontent.com/assets/978192/12436655/8345ebd8-bf17-11e5-88c7-48b4ac15ee7f.png)

Maybe you could post a gist of your *Main.js* file where you are trying to log the props.
Also a screen shot of your dev console could help here.

**Comment by caynan** [comment]
I'm sorry for the delay in answering back, here it's the [gist](https://gist.github.com/caynan/59ad5e6e4ff2995e6203) of the `Main.js` file and the console output.
![screenshot at jan 21 09-55-35](https://cloud.githubusercontent.com/assets/1858994/12489430/22881352-c025-11e5-848c-278474d3164d.png)

Please let me know if you need any other information.

**Comment by stylesuxx** [comment]
Interesting, this is console.log from a fresh scaffolding? It is missing all the warnings it should have (see my screenshot). Can you please share your whole project? I'd like to run it locally - something major seems to be broken here,....

**Comment by caynan** [comment]
sure, you can see the project [here](https://github.com/caynan/react-webpack-redux). I really appreciate your responsiveness and I'll gladly help you solve this problem.

**Comment by stylesuxx** [comment]
Alright, I see what is going on here. My PR for the template got merged, and I did not update this generator accordingly and had everything linked locally, so I did not got the error on my side :-/

The temporary fix for you would be to basically copy the contents of /src/components/run.js to /src/index.js and adapt the paths. This is what your index.js should look like:

```JS
import React from 'react';
import { render } from 'react-dom';
import { Provider } from 'react-redux';
import configureStore from './stores';
import App from './containers/App';

const store = configureStore();

render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById('app')
);
```

I will update the generator accordingly - this may take a day or two.

Thank you for helping me find the problem :-)
